Rear Rider/Rear Rider/ConfigurationOptions/OptionsView.swift

```diff
@@ -21,56 +21,79 @@ extension Binding {
 }
 
 struct OptionsView: View {
-    @ObservedObject var conf = UserConfig()
+    @EnvironmentObject var conf: UserConfig
     @State var confAudio: String = ""
-    @State var confLightPattern: String = ""
+    @State var confLightPattern: Int = 1
     @State var confLightBrightness: Int = 1
     @State var confLightColor: Color = .white
     
+    @State var configChanged = false
+    
     let audioFiles: [ConfigOptions.AudioFile] = ConfigOptions.AudioFile.allCases
     let lightPatterns: [ConfigOptions.LightPattern] = ConfigOptions.LightPattern.allCases
     let lightBrightness: [ConfigOptions.LightBrightness] = ConfigOptions.LightBrightness.allCases
     
     let alert = RearRiderAlerts()
     
     // need to wrap these in this init to get access to self for the conf variable and set these
-    init() {
-        self._confAudio = State(wrappedValue: conf.audioFile)
-        self._confLightPattern = State(wrappedValue: conf.lightPattern)
-        self._confLightBrightness = State(wrappedValue: conf.lightBrightness)
-        self._confLightColor = State(wrappedValue: Color.fromRGBString(rgbString: conf.lightColor))
-    }
+//    init() {
+//        _confAudio = State(wrappedValue: conf.audioFile)
+//        _confLightPattern = State(wrappedValue: conf.lightPattern)
+//        _confLightBrightness = State(wrappedValue: conf.lightBrightness)
+//        _confLightColor = State(wrappedValue: Color.fromRGBString(rgbString: conf.lightColor))
+//    }
```

Comment:
```suggestion
```

Reply:
Also remove the comment saying why this init was needed in the first place if you're removing the init

Reply:
I'm not sure about the init function because I couldn't get it to work in the same way I was doing the other things. If you don't have an idea how to fix it I can remove it.